### PR TITLE
Dividebyzero/147/appointments

### PR DESCRIPTION
**Pull request by Lariiii**

### CODE REVIEW

File: spec/views/events/schedule.html.erb_spec.rb [hunk 1]
```diff
@@ -3,22 +3,34 @@
...
     assign(:event, @league)
     assign(:schedule_type, 'league')
     assign(:matches, @league.matches)
   end
 
+  #before :each do
+  #  @league = FactoryBot.create(:league, :with_matches,
+  #                              deadline: Date.parse('23.12.2017'),
+  #                              startdate: Date.parse('24.12.2017'),
+  #                              enddate: Date.parse('31.12.2017'),
```
Comment (nikriek) [inline spec/views/events/schedule.html.erb_spec.rb]: Why is this commented out? :)

File: app/views/events/_schedule_league.html.erb [hunk 2]
```diff
@@ -50,6 +50,11 @@
         <%= f.submit t('helpers.links.save'), :class => 'btn btn-default btn-xs' %>
       <% end %>
     </td>
+    <td>
+      <!-- 32400 seconds so it is 09:00 as default time, otherwise it would be 00:00 -->
+      <%= date = (match.start_time_date+match.gameday-1).to_s(:day_and_month) %>
```
Comment (nikriek) [inline app/views/events/_schedule_league.html.erb:55]: Why are you assigning a var?

File: db/migrate/20180121144002_add_start_time_to_match.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddStartTimeToMatch < ActiveRecord::Migration[5.1]
+  def change
+    add_column :matches, :start_time, :datetime
```
Comment (nikriek) [inline db/migrate/20180121144002_add_start_time_to_match.rb]: yeeah

File: app/views/events/_schedule_league.html.erb (hunk 2 above)
Comment (Lariiii) [reply app/views/events/_schedule_league.html.erb:55]: Variable is needed for issue #148

**Review by nikriek — APPROVED** [review]

**Pull request merged**